packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.styles.ts

```diff
@@ -113,18 +115,18 @@ export const getStyles = (props: IDetailsHeaderStyleProps): IDetailsHeaderStyles
         cursor: 'default',
         userSelect: 'none',
         selectors: {
-          '&:hover $check': {
+          [`&:hover .${classNames.check}`]: {
```

Comment:
I wonder if we should unconditionally use global class names in places like this (even if global class names have been disabled by the theme)? Otherwise disabling global class names will totally break a lot of the styling.